CK3toEU4/Source/EU4World/Country/Country.cpp

```diff
@@ -885,6 +885,24 @@ void EU4::Country::setPrimaryCulture(const std::string& culture)
 		details.heir.culture = culture;
 }
 
+//All cultures that make up more than 33% of a countries dev will start the game as accepted
+void EU4::Country::setAcceptedCultures()
+{
+	std::map<std::string, int> cultureDevelopment; // culture, development
+	int substantialDev = 0;
+	for (const auto& province: provinces)
+	{
+		substantialDev += province.second->getDev();
+		cultureDevelopment[province.second->getCulture()] += province.second->getDev();
+	}
+	substantialDev /= 3;
```

Comment:
Integer truncation could lead to the occasional extra accepted culture:
If substantialDev was 31, it would become 10 instead of 10 1/3, and a culture with 10 development would become accepted when it should be rejected.